LICENSE

```diff
@@ -1,6 +1,6 @@
 MIT License
 
-Copyright (c) 2017 Veronika Romashkina
```

Comment:
I think here should be `2017-2018`. Also, we can change `MIT` LICENCE here to `MPL-2.0`